requirements.txt

```diff
@@ -0,0 +1,2 @@
+# generated from manifests external_dependencies
```

Comment:
This should no be created, as `lxml` is already required by standard odoo

Reply:
Thanks!